pkg/synchronizer/naisjob_synchronizer.go

```diff
@@ -147,13 +147,14 @@ func (n *Synchronizer) PrepareNaisjob(naisjob *nais_io_v1.Naisjob) (*Rollout, er
 		return nil, fmt.Errorf("query existing namespace: %s", err)
 	}
 
-	// Assert that CNRM annotations are set on namespaces when CNRM support is enabled
-	if naisjob.Spec.GCP != nil && (naisjob.Spec.GCP.SqlInstances != nil || naisjob.Spec.GCP.Permissions != nil) {
-		if val, ok := namespace.Annotations["cnrm.cloud.google.com/project-id"]; ok {
-			rollout.SetGoogleTeamProjectId(val)
-		} else {
+	if naisjob.Spec.GCP != nil {
+		// App requests gcp resources, verify we've got a GCP team project ID
```

Comment:
Trenger vi disse to kommentarene? Sier ikke de bare det samme som koden?

Reply:
Kan være tilbøyelig til å si meg enig med deg om dette for den første kommentaren.

Den andre vil jeg definitivt påstå er "tribal knowledge" som ikke er tydelig for noen som ikke er kjent med naiserator/clusterene våre fra før av - så der er jeg strengt uenig.

Reply:
Da syns jeg vi skal endre feilmeldingen, fordi det er den brukerne våre ser. Så ikke noen grunn til at den skal være så kryptisk at vi faktisk trenger å kommentere med ekstra informasjon.